server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/model/message/view/MessageFastView.java

```diff
@@ -89,7 +89,6 @@ public MessageFastView build() {
         public void checkState() {
             super.checkState();
             Preconditions.checkState(preview != null, "'preview' is mandatory");
-            Preconditions.checkState(hasAttachment.isPresent(), "'hasAttachment' should be present");
```

Comment:
Why? In which case is it acceptable to not specify if the message has an attachment or not?

Reply:
it's consequence of https://github.com/linagora/james-project/pull/2941#discussion_r353038244